Failing test caused by the grid debouncer fix

The following test started to fail after merging https://github.com/vaadin/vaadin-grid/commit/38a01c5ab889cf088e76a3e675ee94fe60a273b8

```
packages/vaadin-grid-pro/test/edit-column-renderer.test.js:

 ❌ edit column renderer > custom edit mode renderer > should close editor and update value when scrolling edited cell out of view
      AssertionError: expected <input> to be falsy
```

See https://github.com/web-padawan/components-monorepo/pull/37

--- a/test/edit-column-renderer.test.js
+++ b/test/edit-column-renderer.test.js
@@ -238,7 +238,10 @@ describe('edit column renderer', () => {
     });
 
     it('should close editor and update value when scrolling edited cell out of view', () => {
+      grid.items = null;
       grid.items = Array.apply(null, { length: 30 }).map(() => Object.assign({}, createItems()[0]));
+      flushGrid(grid);
+
       cell = getContainerCell(grid.$.items, 0, 0);
       column.editModeRenderer = function (root) {
         root.innerHTML = '<input>';

